src/vit-testing/vitup/src/mode/mock/rest/control/handlers.rs

```diff
@@ -0,0 +1,267 @@
+use crate::config::{Config, SnapshotInitials};
+use crate::mode::mock::rest::reject::GeneralException;
+use crate::mode::mock::{ContextLock, FragmentRecieveStrategy, LedgerState, NetworkCongestionMode};
+use crate::mode::service::manager::file_lister::dump_json;
+use jortestkit::web::api_token::{APIToken, APITokenManager, TokenError};
+use vit_servicing_station_lib::db::models::funds::Fund;
+use vit_servicing_station_lib::v0::result::HandlerResult;
+use warp::{Rejection, Reply};
+
+pub async fn logs_get(context: ContextLock) -> Result<impl Reply, Rejection> {
+    let context_lock = context.lock().unwrap();
+    Ok(HandlerResult(Ok(context_lock.logs())))
+}
+
+pub async fn logs_clear(context: ContextLock) -> Result<impl Reply, Rejection> {
+    let mut context_lock = context.lock().unwrap();
+    context_lock.clear_logs();
+    Ok(warp::reply())
+}
+
+pub async fn file_lister_handler(context: ContextLock) -> Result<impl Reply, Rejection> {
+    let context_lock = context.lock().unwrap();
+    Ok(dump_json(context_lock.working_dir())?).map(|r| warp::reply::json(&r))
```

Comment:
how is the context set? Where is the working dir defined?

Reply:
working dir is set in configuration. Context is set based on configuration and represent mock state (ledger/vit-ss data)